todo_app/data/session_items.py

```diff
@@ -53,6 +53,20 @@ def add_item(title):
 
     return item
 
+# TODO: Finish definition function
+def remove_item(item):
```

Comment:
Started work on remove_items function, but not sure how to resolve the recursion issues.